src/functions.js

```diff
@@ -73,19 +54,7 @@ import "./functions/font";
 import "./functions/accent";
 
 // Horizontal stretchy braces
-defineFunction("horizBrace", [
-    "\\overbrace", "\\underbrace",
-], {
-    numArgs: 1,
-}, function(context, args) {
-    const base = args[0];
-    return {
-        type: "horizBrace",
-        label: context.funcName,
-        isOver: /^\\over/.test(context.funcName),
-        base: base,
-    };
-});
+import "./functions/horizBrace";
```

Comment:
👍
I think we can remove new lines between `import` statements as they are single line.

Reply:
Done.
Also, as an additional commit at the end, I remove intermediate comments (these appear inconsistently only on an arbitrary subset) and alphabetized the whole list of functions/ imports. I can undo this last part if you prefer.